### ISSUE TITLE
Gut & Günstig vs. Edeka in OCR

### ISSUE THREAD

**Issue by hangy** [openfoodfacts/robotoff#64]
[Gut & Günstig](https://de.openfoodfacts.org/cgi/search.pl?action=process&tagtype_0=brands&tag_contains_0=contains&tag_0=Gut%20%26%20G%C3%BCnstig&sort_by=unique_scans_n&page_size=20&axis_x=energy&axis_y=products_n&action=display) is brand sold in [Edeka](https://de.openfoodfacts.org/cgi/search.pl?action=process&tagtype_0=brands&tag_contains_0=contains&tag_0=Edeka&sort_by=unique_scans_n&page_size=20&axis_x=energy&axis_y=products_n&action=display) stroes. As it's produced for Edeka, this is mentioned on the packaging a lot. Robotoff sees it and matches the [regex](https://github.com/openfoodfacts/robotoff/blob/c89c020e9260a98a17724e87d85acf141a2b6374/data/ocr/regex_brands.txt#L42) to Edeka.

It could be useful to not suggest users to add the "Edeka" brand to the products if "Gut & Günstig" is present anywhere on the product (possibly including other images for the same product).

**Comment by hangy** [comment]
Same for Milbona/LIDL: https://de.openfoodfacts.org/produkt/4056489018407/fettarmer-kefir-mild-milbona

**Comment by raphael0202** [comment]
I think we want "mother" brands to be included as well. @teolemon Can you confirm?

**Comment by hangy** [comment]
I don't think that would be useful (in ProductOpener). For example, it could drastically skew averages of the "premium" Edeka products in cases where similar products are sold under the child discount brand Gut & Günstig.

**Comment by hangy** [comment]
cc @stephanegigandet 🙂
